metadata/report/nacos/report.go

```diff
@@ -39,19 +38,19 @@ import (
 )
 
 func init() {
-	ftry := &nacosMetadataReportFactory{}
+	mf := &nacosMetadataReportFactory{}
 	extension.SetMetadataReportFactory("nacos", func() factory.MetadataReportFactory {
-		return ftry
+		return mf
 	})
```

Comment:
Combine

Reply:
I have combined theres lines.

Reply:
This is different.
If the ftry is outside the function, it means that we always return the same instance.
But with this change, it means that we create an instance of factory.
Actually, the MetadataReportFactory should be singleton.
So you can not combine these codes.

Reply:
@hxmhlt @flycash I keep this code for singleton.